crates/rooch-framework/sources/coin_entry.move

```diff
@@ -0,0 +1,20 @@
+module rooch_framework::coin_entry {
+    use rooch_framework::account;
+    use moveos_std::storage_context::StorageContext;
+    use rooch_framework::coin;
+
+    /// Transfer `amount` of coins `CoinType` from `from` to `to`.
+    /// This public entry function requires the `CoinType` to have `key` and `store` abilities.
+    public entry fun transfer_entry<CoinType: key + store>(
```

Comment:
Migrate all coin module entry functions to `coin_entry`?  or rename the `coin_entry` module to `tranfer` module? `transfer_entry` => `transfer`.

@baichuan3

Reply:
I also prefer the alone transfer module.

And there is another problem. There is another `transfer_extend` method in the coin module. It does not implicitly create an account and needs to be commented.

Reply:
👍， we can also provide transfer NFT in the transfer module.

Reply:
> And there is another problem. There is another `transfer_extend` method in the coin module. It does not implicitly create an account and needs to be commented.

The `transfer` and `transfer_extend` in the coin module can not auto-create the `to` account, as #823 mentions. We can add documents to describe it. If a developer invokes `coin::transfer` in the contract, they should check the 'to' account before invoking it.

Reply:
What's the conclusion regarding this?

Reply:
As your solution. @bytedeveloperr 

1. `coin_entry.move` to `transfer.move`.
2. `transfer_entry` function to `transfer_coin` function. `transfer_coin` is better than `coin_transfer`?

In the future, we can add `transfer_nft` to the `transfer` module.

Reply:
Alright, That makes sense. Thank you